src/main/java/com/ericsson/event/translator/CloudEventHandlerConfiguration.java

```diff
@@ -0,0 +1,38 @@
+/*
+    Copyright (C) 2023 Nordix Foundation.
+    For a full list of individual contributors, please see the commit history.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+          http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    SPDX-License-Identifier: Apache-2.0
+*/
+package com.ericsson.event.translator;

```

Comment:
Why is this com.ericsson? Could it instead be io.github.eiffel-community? Or should it be org.nordix since it's contributed from Nordix?

```suggestion
package io.github.eiffel-community.translator-cdevents;

```

or 

```suggestion
package org.nordix.eiffel.translator-cdevents;

```

Reply:
The relationship with nordix is going to change in the next few months 
could we use 
package tech.est.eiffel.translator-cdevents;

as est is the company that it making the contribution. The copyright is still with nordix.

Reply:
That's fine with me

Reply:
Update the package name with `tech.est.eiffel.translator.cdevents`